stable/external-dns/templates/deployment.yaml

```diff
@@ -32,6 +32,16 @@ spec:
           {{- range $key, $value := .Values.extraArgs }}
             - --{{ $key }}={{ $value }}
           {{- end }}
+          env:
+          {{- if and .Values.cloudflare.api_key .Values.cloudflare.email }}
```

Comment:
Nit: indent the `if` & `end` statements one level less than the block they wrap.